src/components/TextInput/TextInput.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import colors from '../../theme/colors.scss';
+
+import './TextInput.scss';
+
+const prefixCls = 'kai-text-input';
+
+class TextInput extends React.PureComponent {
```

Comment:
Should be `.jsx` instead of just `.js`

Reply:
Ah yes, this was a side effect of my experimenting with Rollup. It wouldn't resolve `.jsx` files. I've got that sorted now though, so will rename this back. 👍